### PR TITLE
♻️ Add utm mixin for app download pages

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

**Comment by nwingt** [comment]
I suggest adding new computed `utmProps` in `utmMixin` to return `{ utmSource, utm... }`, then u can use `v:bind="utmProps"` for short

**Comment by williamchong** [comment]
How can I handle cases where one of the field have default value? e.g. `utmMedium: this.utmMedium || 'banner'`

**Comment by nwingt** [comment]
then I suggest to add method in mixin `getUtmProps({  utmMedium: 'banner' })`

File: src/components/CreatorsPage/CreatorsPage.vue [hunk 1]
```diff
@@ -88,9 +88,7 @@
           AppLogo
           AppDownloadBadges.mt-16(
             type="single"
-            :utm-campaign="utmCampaign || 'creators_page'"
-            :utm-source="utmSource"
-            :utm-medium="utmMedium"
+            v-bind="this.getUtmProps({ utmMedium: 'creators_page' })"
```
Comment (nwingt) [inline src/components/CreatorsPage/CreatorsPage.vue:91]: can omit `this.

File: src/pages/getapp.vue [hunk 2]
```diff
@@ -7,9 +7,7 @@
     main.page-content
       GetAppPage(
         :from="$route.query.from"
-        :utm-campaign="utmCampaign"
-        :utm-source="utmSource"
-        :utm-medium="utmMedium"
+        v-bind="this.getUtmProps()"
```
Comment (nwingt) [inline src/pages/getapp.vue:10]: Can omit `this.`

**Review by nwingt — CHANGES_REQUESTED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**